.github/workflows/release-docker.yaml

```diff
@@ -0,0 +1,63 @@
+name: Release Docker Images
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    secrets:
+      DOCKER_HUB_USERNAME:
+        required: true
+      DOCKER_HUB_PASSWORD:
+        required: true
+
+defaults:
+  run:
+    working-directory: .
+
+jobs:
+  build-image-matrix:
+    if: github.repository_owner == 'mosaicml'
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+      - uses: actions/checkout@v3
+      - id: set-matrix
+        run: |
+          # Install yaml dependency
+          pip install pyyaml
+
+          # Generate build matrix
+          BUILD_MATRIX=$(python .github/bin/gen_docker_matrix.py docker/build_matrix.yaml)
+          echo $BUILD_MATRIX >> $GITHUB_OUTPUT
+
+  stage-docker-build:
+    needs: build-image-matrix
+    uses: ./.github/workflows/docker-configure-build-push.yaml
+    strategy:
+      fail-fast: true
+      matrix: ${{ fromJSON(needs.build-image-matrix.outputs.matrix) }}
+    name: ${{ matrix.IMAGE_NAME }}
+    with:
+      build-args: |
+        AWS_OFI_NCCL_VERSION=${{ matrix.AWS_OFI_NCCL_VERSION }}
+        BASE_IMAGE=${{ matrix.BASE_IMAGE }}
+        CUDA_VERSION=${{ matrix.CUDA_VERSION }}
+        MOFED_VERSION=${{ matrix.MOFED_VERSION }}
+        PYTHON_VERSION=${{ matrix.PYTHON_VERSION }}
+        PYTORCH_VERSION=${{ matrix.PYTORCH_VERSION }}
+        TORCHTEXT_VERSION=${{ matrix.TORCHTEXT_VERSION }}
+        TORCHVISION_VERSION=${{ matrix.TORCHVISION_VERSION }}
+      context: ./docker
+      image-name: ${{ matrix.IMAGE_NAME }}
+      image-uuid: ${{ matrix.UUID }}
+      push: false
```

Comment:
I didn't quite understand. Isn't the "release" pipeline supposed to push docker images?

Reply:
Also confused about this

Reply:
Good catch, had that as false before bc I didn't want to accidentally push new images.